src/App.js

```diff
@@ -10,7 +11,10 @@ import Header from "./components/Header.js";
 import Footer from "./components/Footer.js";
 
 export default function App() {
+  const history = useHistory();
   const [sudokuDifficulty, setSudokuDifficulty] = useState("");
+  const [isRoomSelected, setIsRoomSelected] = useState(false);
+
   useEffect(() => {
     let currentDifficulty = JSON.parse(
       localStorage.getItem("currentDifficulty")
```

Comment:
you could write 

const currentDifficulty = JSON.parse(localStorage.getItem("currentDifficulty")) || "???";

and save yourself the the if statement below